lib/highline.rb

```diff
@@ -989,12 +989,12 @@ def get_response(  )
   # instead.  This is to support any special handling for the final sequence.
   #
   def page_print( output )
-    lines = output.scan(/[^\n]*\n?/)
+    lines = output.lines.to_a
```

Comment:
With Ruby >= 2.0.0 String#lines returns an Array.
But when 1.9.3 is the case, String#lines returns an Enumerator.
See: 
* http://ruby-doc.org/core-2.0.0/String.html#method-i-lines
* http://ruby-doc.org/core-1.9.3/String.html#method-i-lines

That was the "why" tests were failing only on 1.9.3.
See travis log: 
* https://travis-ci.org/abinoam/highline/jobs/84543775
```
Error: test_paging(TestHighLine): NoMethodError: undefined method `size' for #<Enumerator:0x000000031c79f8>
/home/travis/build/abinoam/highline/lib/highline.rb:993:in `page_print'
/home/travis/build/abinoam/highline/lib/highline.rb:724:in `format_statement'
/home/travis/build/abinoam/highline/lib/highline.rb:621:in `say'
/home/travis/build/abinoam/highline/test/tc_highline.rb:972:in `test_paging'
```